api/serverV2_test.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/testutil"
+)
+
+func TestServerV2Start(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, _ := createServerV2(cfg, false)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	err := svr.Start()
+	require.NoError(err)
+
+	time.Sleep(3 * time.Second)
```

Comment:
does a short time like 300ms work?

Reply:
same flaky problem as https://github.com/iotexproject/iotex-core/pull/2898
pls fix this problem before merge

Reply:
ok